kazu/conf/SynonymGeneration/default.yaml

```diff
@@ -58,37 +58,37 @@ meddra:
       include_greek: true
     - _target_: kazu.ontology_preprocessing.synonym_generation.TokenListReplacementGenerator
       token_lists_to_consider:
-        -  - anomalous
```

Comment:
not sure whats changed here? is this black formatting?

Reply:
ah I see it's the indent level

Reply:
yeah sorry when I save in VSCode it has some setting which changes it and produces unneeded diff. It is probably the YAML extension I have